packaging/os/rhn_register.py

```diff
@@ -119,7 +119,8 @@
 try:
     import up2date_client
     import up2date_client.config
-except ImportError, e:
+except ImportError:
+    e = get_exception()
```

Comment:
Yeah, I was wondering if it would work or not when I wrote the patch, but didn't dig much. Another solution is just to not get the exception in a variable and just show a simple generic error message.